pkgdepend doesn't like the first line of a Python script to be:
'#!/usr/bin/env python' so turn it into '#!/usr/bin/python2.7'

Note that there is a COMPONENT_POST_BUILD_ACTION in the component
Makefile to then turn them into #!/usr/bin/python2.6 for when we
do the Python 2.6 build.

Yes we do need the Control-M's in here. Initially the Python source
files have them, and there is a COMPONENT_PRE_BUILD_ACTION in the
component Makefile to strip them out, otherwise pkgdepend gets confused.

--- cssutils-0.9.6/src/cssutils/tokenize2.py.orig	2009-12-04 19:57:40.769947686 +0800
+++ cssutils-0.9.6/src/cssutils/tokenize2.py	2009-12-04 19:58:06.792081272 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 # -*- coding: utf-8 -*-
 """New CSS Tokenizer (a generator)
 """
--- cssutils-0.9.6/src/cssutils/serialize.py.orig	2009-12-07 10:17:54.826455529 +0800
+++ cssutils-0.9.6/src/cssutils/serialize.py	2009-12-07 10:19:33.138080643 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 # -*- coding: utf-8 -*-
 """cssutils serializer"""
 __all__ = ['CSSSerializer', 'Preferences']
@@ -948,4 +948,4 @@
                 out.append(val, None, space=False)
                 
             return out.value() 
-            
\ No newline at end of file
+            
--- cssutils-0.9.6/src/cssutils/__init__.py.orig	2009-12-07 10:21:25.702234442 +0800
+++ cssutils-0.9.6/src/cssutils/__init__.py	2009-12-07 10:21:44.667894629 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """cssutils - CSS Cascading Style Sheets library for Python
 
     Copyright (C) 2004-2009 Christof Hoeke
--- cssutils-0.9.6/src/cssutils/scripts/csscombine.py.orig	2009-12-07 10:22:27.582839434 +0800
+++ cssutils-0.9.6/src/cssutils/scripts/csscombine.py	2009-12-07 10:22:39.860764499 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """Combine sheets referred to by @import rules in a given CSS proxy sheet
 into a single new sheet.
 
@@ -96,4 +96,4 @@
 
 
 if __name__ == '__main__':
-    sys.exit(main())
\ No newline at end of file
+    sys.exit(main())
--- cssutils-0.9.6/src/cssutils/scripts/cssparse.py.orig	2009-12-07 10:23:30.623787121 +0800
+++ cssutils-0.9.6/src/cssutils/scripts/cssparse.py	2009-12-07 10:23:40.469638156 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """utility script to parse given filenames or string
 """
 __docformat__ = 'restructuredtext'
--- cssutils-0.9.6/src/cssutils/scripts/csscapture.py.orig	2009-12-07 10:24:13.568541655 +0800
+++ cssutils-0.9.6/src/cssutils/scripts/csscapture.py	2009-12-07 10:24:27.094792825 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """Retrieve all CSS stylesheets including embedded for a given URL.
 Retrieve as StyleSheetList or save to disk - raw, parsed or minified version.
 
--- cssutils-0.9.6/src/cssutils/errorhandler.py.orig	2009-12-07 10:25:31.225803334 +0800
+++ cssutils-0.9.6/src/cssutils/errorhandler.py	2009-12-07 10:25:41.199601897 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """cssutils ErrorHandler
 
 ErrorHandler
--- cssutils-0.9.6/src/cssutils/parse.py.orig	2009-12-07 10:26:23.218622184 +0800
+++ cssutils-0.9.6/src/cssutils/parse.py	2009-12-07 10:26:32.968490941 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """A validating CSSParser"""
 __all__ = ['CSSParser']
 __docformat__ = 'restructuredtext'
--- cssutils-0.9.6/src/encutils/__init__.py.orig	2009-12-07 10:27:04.131362666 +0800
+++ cssutils-0.9.6/src/encutils/__init__.py	2009-12-07 10:27:17.697342850 +0800
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """encutils - encoding detection collection for Python
 
 :Version: 0.9
@@ -682,4 +682,4 @@
 
 if __name__ == '__main__':
     import pydoc
-    pydoc.help(__name__)
\ No newline at end of file
+    pydoc.help(__name__)
--- cssutils-0.9.6/src/cssutils/codec.py.orig	2012-08-07 06:45:29.982973309 -0700
+++ cssutils-0.9.6/src/cssutils/codec.py	2012-08-07 06:45:41.155738434 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 """Python codec for CSS."""
 __docformat__ = 'restructuredtext'
 __author__ = 'Walter Doerwald'
